docs/modules/release-notes/pages/all-releases/4_14.adoc

```diff
@@ -0,0 +1,117 @@
+= Kobiton 4.14 release notes
+:navtitle: Kobiton 4.14 release notes
+
+_August 2, 2024_
+
+== New OS support
+
+Kobiton 4.14 adds support for the following:
+
+* iOS/iPadOS 18 Beta 3
+* Xcode 16 Beta
+
+== Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced iOS 17, they rolled out new connectivity protocols that impacted how iOS 17 devices communicated with solutions like Kobiton. These protocols required special network configurations for Kobiton Lightning Mode deployments that were not always possible for customers to implement.
+
+With iOS 17.4, Apple introduced a new communication protocol that significantly enhances connections between iOS devices and the Kobiton platform. We implemented this innovative protocol with this release. This protocol eliminates the need for the special Lightning Mode networking changes introduced with iOS 17 for iOS 17.4+ devices, simplifying your networking.
+
+[IMPORTANT]
+====
+
+* iOS 17.0-17.3 devices continue to require special networking to support Lightning mode, so we recommend those devices are hosted on a separate Mac mini in Standard mode.
+
+====
+
+[NOTE]
+====
+
+* iOS 17.4 and later devices still need to be plugged into the Mac mini to establish trust before connecting to the GEM.
+* Metrics is not available for iOS 17.0 to 17.3 devices.
+* virtualUSB is not available for iOS devices.
+* Android 7 to 15, iOS 14 to 16 are still supported by both Standard and Lightning mode.
+
+====
+
+== SSO: allow non-SSO login for specific users in SSO only organizations
+
+It is now possible to specify some users in SSO only organizations to use the Kobiton username and password to log in.
+
+In SSO settings, when the option _Enforce users to login to Kobiton only through SSO_ is turned on, there is now a new field *_Choose users who are allowed to login without SSO_*. Search for and add existing Kobiton users to this field to allow them to log in using Kobiton username and password.
+
+When _Enforce users to login to Kobiton only through SSO_ is turned on, the password of all existing users are cleared from database. Therefore, the users that are allowed to log in using Kobiton username and password must use *Forgot password* to create a new password.
+
+
+== Indicator for Lightning mode devices in device lists
```

Comment:
Should lists be singular? 
Since it's a page name = should we capitalize "Device List"

Reply:
updated